roles/hosted_engine_setup/tasks/bootstrap_local_vm/03_engine_initial_tasks.yml

```diff
@@ -54,16 +54,24 @@
       - "OVESETUP_DWH_DB/password=str:{{ he_dwh_db_password }}"
     when: he_dwh_db_password is defined
   - name: Import OpenSCAP task
-    import_tasks: apply_openscap_profile.yml
+    import_tasks: ../apply_openscap_profile.yml
     when: he_apply_openscap_profile|bool
   - name: Include before engine-setup custom tasks files for the engine VM
     include_tasks: "{{ item }}"
     with_fileglob: "hooks/enginevm_before_engine_setup/*.yml"
     register: include_before_engine_setup_results
-  - debug: var=include_before_engine_setup_results
+  - name: Pause the execution to allow the user to configure the bootstrap engine VM
+    block:
+      - name: Allow the user to connect to the bootstrap engine and change configuration
+        debug:
+          msg: >-
+            You can now connect to the bootstrap engine using the temporary IP address -
```

Comment:
Perhaps 'using ssh as root, from this host, to the temporary...'. I suppose it's only from this host, right?